source/clear-linux/guides/maintenance/kernel-development.rst

```diff
@@ -0,0 +1,462 @@
+      #
+      %patch0501 -p1
+      %patch0502 -p1
+
+      #Serie1.patch.start
+      %patch1001 -p1
+      #Serie1.patch.end
+
+      %patch2001 -p1
+
+
+#. Commit and save the changes to the RPM SPEC file.
+
+
+
+Modify kernel boot parameters
+=============================
+The kernel boot options are passed from the bootloader to the kernel with 
+command-line parameters. 
+
+While temporarily changes can be made to kernel parameters on a running 
+system or on a during boot, you can also modify the default parameters that 
+are persistent and distributed with a customized kernel.
+
+
+#. Open the kernel :file:`cmdline` file in an editor.
+
+   .. code-block:: bash
+
+      $EDITOR cmdline
+
+
+#. Make any desired change to the kernel parameters. 
+   For example, you can remove the :command:`quiet` parameter to see more 
+   verbose output of kernel log messages during the boot process.
+
+
+#. Commit and save the changes to the :file:`cmdline` file.
+
+See the 
+`Kernel parameters documentation`_ for a list of available parameters. 
+
+
+
+Build and install the kernel
+****************************
+After changes have been made to the kernel source and RPM SPEC file, 
+the kernel is ready to be compiled and packaged into an RPM.
+
+#. Start the compilation process by issuing the :command:`make build` command. 
+   This process is typically resource intensive and will take a while.
+
+   .. code-block:: bash
+      
+      make build
+
+
+#. The result will be multiple :file:`.rpm` files in the :file:`rpms` 
+   directory as output. 
+
+   .. code-block:: bash
+      
+      ls rpms/
+
+   The kernel RPM will be named :file:`linux<NAME>-<VERSION>-<RELEASE>.x86_64.rpm`
+
+
+#. The kernel RPM file can be input to the `mixer tool`_ to create a custom 
+   bundle and mix of |CL|.
+
+   
+
+Alternatively, the kernel RPM bundle can be installed manually on a local 
+machine for testing. This approach works well for individual development or 
+testing. For a more scalable and customizable approach, consider using the 
+`mixer tool`_ to provide a custom kernel with updates.
+
+1. Install the kernel onto the local system by extracting the RPM with the 
+   :command:`rpm2cpio` command.
+
+   .. code-block:: bash
+
+      sudo rpm2cpio linux<NAME>-<VERSION>-<RELEASE>.x86_64.rpm | (cd /; sudo cpio -i -d -u -v);
+
+
+#. Update the |CL| boot manager using :command:`clr-boot-manager` and reboot.
+
+   .. code-block:: bash
+
+      sudo clr-boot-manager list-kernels
+      sudo clr-boot-manager set-kernel org.clearlinux.<TARGET>.<VERSION>-<RELEASE>
+
+      sudo reboot
+
+
+#. After a reboot, verify the customized kernel is running.
+
+   .. code-block:: bash
+
+      uname -a
+
+
+
+Related topics
+**************
+
+* :ref:`kernel-modules`
+* :ref:`mixer`
+
+.. _`kernels available in Clear Linux`: https://clearlinux.org/documentation/clear-linux/reference/compatible-kernels
+.. _`on GitHub`: https://github.com/clearlinux/distribution 
```

Comment:
Here it is better to use

https://github.com/clearlinux/distribution/issues/new/choose